### ISSUE TITLE
[NoiseEngine] more documentation (Trac #1229)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1223]
I think a link to a presentation is not the best solution and the doxygen documentation does not really answer what this project does and how. 

Migrated from https://code.icecube.wisc.edu/ticket/1229
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "I think a link to a presentation is not the best solution and the doxygen documentation does not really answer what this project does and how. ", 
    "reporter": "kkrings", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[NoiseEngine] more documentation", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2015-08-19T21:17:59", 
    "milestone": "", 
    "owner": "mjl5147", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150820T12:35:45: 
* mjl5147 changed owner from "mlarson" to "mjl5147"
* mjl5147 changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20150820T21:40:58: kkrings commented:

> Release notes should be linked to the documentation's front page.

**Comment by icecube-trac** [comment]
Trac update at 20160219T17:04:35: 
* mjl5147 changed resolution from "" to "fixed"
* mjl5147 changed status from "assigned" to "closed"
* mjl5147 commented:

> In changeset 142115:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="142115"
> Removing the I3DOMLaunch templating (which shouldn't be used anyway: the InIceRawData doesn't exist in 2012+ anyway and the module has always officially been run on I3RecoPulses), added documentation to close #1223, added a couple of python test scripts, and fixed the tray segment to close #1224. Removed the unused serialization from the HitPair class as well
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
